third_party/move/move-compiler-v2/transactional-tests/tests/constants/large_vectors.optimize-no-simplify.exp

```diff
@@ -10,6 +10,8 @@ comparison between v1 and v2 failed:
 +   │                        ^
 + 
 + 
++ Exiting with bytecode verification errors
```

Comment:
Why are those tests now failing where they succeeded before?

Reply:
This test was failing before, but less noisily.  It now says "Exiting with ..." when exiting.  This is particularly essential in transactional tests in the case of --warnings-as-errors, as we can tell that the compilation is exiting early, but it also seems useful in other cases so I left it in.